internal/handlers/cart.go

```diff
@@ -0,0 +1,67 @@
+package handlers
+
+import (
+	"Cart_Api_New/internal/errorsx"
+	"context"
+	"encoding/json"
+	"errors"
+	"log"
+	"net/http"
+	"strconv"
+	"time"
+)
+
+func (h Handler) CreateCart(w http.ResponseWriter, r *http.Request) {
+	ctx, cancel := context.WithTimeout(r.Context(), time.Second*1)
+
+	defer cancel()
+
+	newCart, err := h.service.CreateNewCart(ctx)
+
+	if err != nil {
+		log.Printf("from h.service.CreateNewCart: %v", err)
+		errorWrite(w, errorsx.InternalServerErr, http.StatusInternalServerError)
+		return
+	}
+
+	err = json.NewEncoder(w).Encode(newCart)
+	if err != nil {
+		log.Printf("from json.NewEncoder: %v", err)
+		errorWrite(w, errorsx.InternalServerErr, http.StatusInternalServerError)
+		return
+	}
+}
+
+func (h Handler) GetCart(w http.ResponseWriter, r *http.Request) {
+	idCartNumber, err := strconv.Atoi(r.PathValue("id"))
+	if err != nil {
+		log.Printf("from strconv.Atoi: %v", err)
+		errorWrite(w, errorsx.InvalidCartIdErr, http.StatusBadRequest)
+		return
+	}
+
+	ctx, cancel := context.WithTimeout(r.Context(), time.Second*1)
```

Comment:
make 1 as const and name it like requestTimeToProcess
move it at the beginning of fuction